docs/FAQ.md

```diff
@@ -93,3 +93,33 @@ quilt3 push --dir path/to/remote-registry my-package # upload Quilt data package
 ### Using Quilt with Reticulate
 The [Reticulate](https://rstudio.github.io/reticulate/) package provides a set of tools
 for interoperability between Python and R by embedding a Python session within your R session.
+
+## How do I delete all of the objects in a data package?
+
+*Very carefully!* In favor of immutability Quilt makes deletion a
+bit tricky. First, note that `quilt3.Package.delete` only deletes the
+_package manifest_, not the *underlying objects*. If you wish to delete
+a package revision *and* its objects, _delete the objects first_.
```

Comment:
I'm confused. Is there a difference between "deleting a manifest" and "deleting a revision"?
If so, what? And which is shown in the example below?

Reply:
Okay, then may I suggest replacing:
> delete a package revision
with
> delete the entire package

Reply:
@drernie Good feedback. Incorporated. Thanks!